tests/MongoDB.Driver.Core.Tests/Core/Clusters/MultiServerClusterTests.cs

```diff
@@ -736,6 +735,8 @@ public void Should_hide_a_seedlist_server_of_the_wrong_type(ClusterConnectionMod
             var description = subject.Description;
             description.Servers.Should().BeEquivalentToWithComparer(GetDescriptions(_firstEndPoint, _thirdEndPoint), _serverDescriptionComparer);
 
+            _capturedEvents.Next().Should().BeOfType<ClusterRemovingServerEvent>();
```

Comment:
Now, since we remove a server when it is not valid for the current cluster type => we see two new remove events